From 2a10b58fad810aca1a041cd37b00241ace9be0c7 Mon Sep 17 00:00:00 2001
From: Marius Vollmer <mvollmer@redhat.com>
Date: Mon, 13 Oct 2014 15:58:46 -0500
Subject: [PATCH 1/4] policy: allow non-local admin sessions to control the
 network (rh #1145646)

(cherry picked from commit 4f950ee5692e57bbc9737720adfd7ad6788dabb7)
---
 policy/org.freedesktop.NetworkManager.policy.in.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/org.freedesktop.NetworkManager.policy.in.in b/policy/org.freedesktop.NetworkManager.policy.in.in
index cb22999..daefd80 100644
--- a/policy/org.freedesktop.NetworkManager.policy.in.in
+++ b/policy/org.freedesktop.NetworkManager.policy.in.in
@@ -58,6 +58,7 @@
     <_description>Allow control of network connections</_description>
     <_message>System policy prevents control of network connections</_message>
     <defaults>
+      <allow_any>auth_admin</allow_any>
       <allow_inactive>yes</allow_inactive>
       <allow_active>yes</allow_active>
     </defaults>
-- 
1.9.3

